link/netkit_test.go

```diff
@@ -5,12 +5,13 @@ import (
 	"sync/atomic"
 	"testing"
 
-	"github.com/cilium/ebpf"
-	"github.com/cilium/ebpf/internal/testutils"
 	"github.com/go-quicktest/qt"
 	"github.com/jsimonetti/rtnetlink/v2"
 	"github.com/jsimonetti/rtnetlink/v2/driver"
-	"golang.org/x/sys/unix"
+
+	"github.com/cilium/ebpf"
```

Comment:
Should we add a linter that checks if imports are formatted correctly? Happy to open an issue!

Reply:
Skipped the issue and just opened https://github.com/cilium/ebpf/pull/1541